api_tests/tests/orderbook.ts

```diff
@@ -7,16 +7,32 @@ import { twoActorTest } from "../src/actor_test";
 import OrderFactory from "../src/actors/order_factory";
 import { sleep } from "../src/utils";
 
-// todo: move test initialisation into single mega function to reduce noise
 describe("orderbook", () => {
+    // direct quote
+    // pair: "BTC/DAI"
+    // position: buy
+    // rate: 100 (1000 wei for 1 satoshi)
+    // amount: 900,000 satoshi BTC
```

Comment:
Please use 'price' and 'quantity' as outlined here: https://3.basecamp.com/4403044/buckets/15109376/messages/2878398232

Reply:
The nectar code base is using rate and amount so maintained consistency with nectar

Reply:
I feel adopting a new terminology should be its own PR, otherwise we will never get this in :grimacing:

Reply:
I agree that the switch to forex terminology should be a separate PR. The source of truth is the basecamp post, all other projects should by [slowly] migrated.